README.md

```diff
@@ -140,24 +140,36 @@ kubectl apply -f https://raw.githubusercontent.com/fluent/fluentbit-operator/mas
 The Fluent Bit section of the Fluent Operator supports different CRI `docker`, `containerd`,  and `CRI-O`. 
 `containerd` and `CRI-O` use the `CRI Log` format which is different with `docker`, they requires additional parser to parse JSON application logs. You should set different `containerRuntime` depending on your container runtime.
 
-If your container runtime is `docker`
+If your container runtime is `docker`:
 
 ```shell
-helm install fluent-operator  --create-namespace -n kubesphere-logging-system charts/fluent-operator/ --set containerRuntime=docker
+helm install fluent-operator  --create-namespace -n kubesphere-logging-system charts/fluent-operator/ 
 ```
 
-If your container runtime is `containerd`
+If your container runtime is `containerd`:
 
 ```shell
 helm install fluent-operator --create-namespace -n kubesphere-logging-system charts/fluent-operator/  --set containerRuntime=containerd
 ```
 
-If your container runtime is `cri-o`
+If your container runtime is `cri-o`:
 
 ```shell
 helm install fluent-operator --create-namespace -n kubesphere-logging-system charts/fluent-operator/  --set containerRuntime=crio
 ```
 
+Install through the online chart link by releases:
+
+```shell
+helm install fluent-operator --create-namespace -n kubesphere-logging-system https://github.com/fluent/fluent-operator/releases/download/v1.0.0-rc.0/fluent-operator.tgz  --set containerRuntime=docker
+```
+
+Install through the online chart link by master branch:
+
+```shell
+helm install fluent-operator --create-namespace -n kubesphere-logging-system https://github.com/fluent/fluent-operator/blob/master/charts/fluent-operator.tgz  --set containerRuntime=docker
```

Comment:
Do we need to specify the runtime for Docker?

Reply:
The default runtime is docker I think.  /cc @wenchajun Can you help answer it?